### ISSUE TITLE
ingest_upload - should be moving, not copying blobs to the DATA bucket

### ISSUE THREAD

**Issue by curlup** [CIMAC-CIDC/cidc-cloud-functions#12]
Now it just copies it:
https://github.com/CIMAC-CIDC/cidc-cloud-functions/blob/db5faec173d5dd176d64eac2c275d7da5bcae9a7/functions/uploads.py#L50
and 
https://github.com/CIMAC-CIDC/cidc-cloud-functions/blob/db5faec173d5dd176d64eac2c275d7da5bcae9a7/functions/uploads.py#L68
leaving an unneeded copy in the UPLOAD bucket.

I suggest we delete it.

**Comment by jacoblurye** [comment]
Yeah, we should think about this. An argument I can think of in favor of not deleting the old copy is to support rollbacks to previous uploads, but I don't know if that's something we need to be able to do.
